### ISSUE TITLE
Separate tegname and tagalias in DB.

### ISSUE THREAD

**Issue by Artexoid** [goldsky/SmartTag#36]
Hi. Can you separate tagname and tagalias in DB. It's need for correct using national languages in tegname and english in tagalias. As long we have to use the national language in URL. This isn't very good...

Thanks!

**Comment by goldsky** [comment]
I don't follow

**Comment by Artexoid** [comment]
Very vary sad. It's good extension.

**Comment by goldsky** [comment]
I don't follow your explanation

**Comment by goldsky** [comment]
I don't understand your description

**Comment by Artexoid** [comment]
Aaa.. Ok ok.. Sorry for my English )

I create new tag 'TAG'.. It insert rew 'smarttag_tags'

id=>1, tag=>TAG

And for use tag i must folow to url (example) site.ru/section/TAG/

BUT if i use russian language (for example i create tag 'Мой Тег'), a will have

id=>1, tag=>Мой Тег

and i must follow to url site.ru/section/Мой%20Тег/ and this isn't good!!!

I propose to add the **alias** field to smarttag_tags

**Comment by Artexoid** [comment]
it's ok description? You understand my idea ?? :-)

**Comment by Artexoid** [comment]
Sorry, I just started to learn English :-)!!!

**Comment by goldsky** [comment]
Ok, now I understand, tks.

I believe it would bring new problem.
SmartTag only to manage the tag, but we need other snippet to filter the
document, lets say getResourceTag.
That snippet has its own logic on getting the resources, which is not an
alias, but rather url decode of the tag.

**Comment by Artexoid** [comment]
Yes yes, i know. I agree with you, and i use custom snippet for get resources by tag, but i'm talking about only identifying tag from url param without urldecode...

This is the aesthetic character.

1. site/section/tag/  (in snippet i see 'my-tag' == 'Мой Тег')
2. site/section/%D0%9C%D0%BE%D0%B9%20%D0%A2%D0%B5%D0%B3/ (in snippet i see '%D0%9C%D0%BE%D0%B9%20%D0%A2%D0%B5%D0%B3' == 'Мой Тег' **but it's very scary!!!**) :-D

1. good url
2. bad url :-)

Alias may be generated automatically use Latin letter always without control panel(control field) for it.
